third_party/system_libraries.BUILD

```diff
@@ -46,12 +46,20 @@ cc_library(
 
 cc_library(
     name = "libtins",
-    srcs = ["usr/lib/libtins.so"],
     linkopts = ["-ltins"],
 )
 
 cc_library(
     name = "libmnl",
-    srcs = ["usr/lib/x86_64-linux-gnu/libmnl.so"],
     linkopts = ["-lmnl"],
 )
+
+cc_library(
+    name = "libpcap",
+    linkopts = ["-lpcap"],
```

Comment:
does this approach work for folly too?

Reply:
I anticipate it does so long as you are using shared lib linking